odoo/fields.py

```diff
@@ -2235,6 +2235,8 @@ class One2many(_RelationalMulti):
 
         The attributes ``comodel_name`` and ``inverse_name`` are mandatory except in
         the case of related fields or field extensions.
+
+        ``related`` is not implemented for One2many and will result in undefined behavior.
```

Comment:
Are you sure about that?

Reply:
https://discord.com/channels/678381219515465750/687339689522364423/745285278138237078

I had forgotten to mention in my notes that _chained_ One2many relationships are not supported through `related`. I will update the description.